compiler/optimizer/OMRSimplifierTableEnum.hpp

```diff
@@ -63,17 +63,29 @@
    dftSimplifier,           // TR::fstore
    dftSimplifier,           // TR::dstore
    astoreSimplifier,        // TR::astore
-   dftSimplifier,           // TR::wrtbar
    dftSimplifier,           // TR::bstore
    dftSimplifier,           // TR::sstore
+   directStoreSimplifier,   // TR::iwrtbar
```

Comment:
why is this one different?

Reply:
The wrtbars are using the same simplifiers as the corresponding stores except the awrtbar which used dftSimplifier in the old implemenation already so I kept the old implmentation.